scripts/skills/perks/perk_rf_kingfisher.nut

```diff
@@ -19,7 +19,18 @@ this.perk_rf_kingfisher <- ::inherit("scripts/skills/skill", {
 	{
 		if (this.m.IsForceEnabled) return true;
 
-		return this.getContainer().hasSkill("actives.throw_net"); // Ignores hidden skills
+		// Ensure that the actor has an offhand item with the throw_net skill
+		local offhandItem = this.getContainer().getActor().getOffhandItem();
+		if (offhandItem == null)
+			return false;
+
+		foreach (skill in offhandItem.getSkills())
+		{
+			if (skill.getID() == "actives.throw_net" && !skill.isHidden())
```

Comment:
Just for some context:
The compatibility issue stems from [a mod](https://www.nexusmods.com/battlebrothers/mods/69) which leaves nets equipped in the offhand after use, and disables (greys out) their ability.
That was the original reason for us to revisit this code.

Unfortunately we can't differentiate the reason for the net being unusable. So the weird behavior with that mod will remain. Until the owner bothers to hide the skill, instead of just disabling it.

Reply:
Isn't the item number limit for Offhand Slot == 1? If yes, then how can that mod keep an item in that slot and still allow the player to equip some other item in that slot? That seems like a bad/dangerous move by that mod.

Reply:
Here is an image on how it looks ingame.
The used net is treated like an empty throwing weapon. It's still a full item that you need to manually unequip/swap with other items

![image](https://github.com/user-attachments/assets/88cf72a1-5652-482a-8e7a-fac7f0928054)